### PR TITLE
[Doc improvement] Update articles for latest TeamsJS (v.2)

### PR DESCRIPTION
**Pull request by JoshuaPartlow**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/location-capability.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9c76fca9ad84dbed1a30f5be53aac97b048ac7d7/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Flocation-capability.md) | 91 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c80034bc-855c-431e-bd0b-aef9f7066a97) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/conversational-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9c76fca9ad84dbed1a30f5be53aac97b048ac7d7/msteams-platform%2Ftabs%2Fhow-to%2Fconversational-tabs.md) | 94 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f68faf63-feb7-4c20-a0a5-3352bfcdf354) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-channel-group-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9c76fca9ad84dbed1a30f5be53aac97b048ac7d7/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-channel-group-tab.md) | 92 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/28344232-2051-41fd-ac35-b52cfafe5c41) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9c76fca9ad84dbed1a30f5be53aac97b048ac7d7/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-personal-tab.md) | 93 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dbe6e3d7-8d3f-4a09-8b8d-c017938a1156) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-tab-pages/configuration-page.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9c76fca9ad84dbed1a30f5be53aac97b048ac7d7/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-tab-pages%2Fconfiguration-page.md) | 93 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/829c5784-1a0d-4718-8062-40e0e55612d7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-tab-pages/content-page.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9c76fca9ad84dbed1a30f5be53aac97b048ac7d7/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-tab-pages%2Fcontent-page.md) | 90 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c59368dc-d14e-4f49-92e6-a834fedc24e7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/tabs-link-unfurling.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9c76fca9ad84dbed1a30f5be53aac97b048ac7d7/msteams-platform%2Ftabs%2Ftabs-link-unfurling.md) | 89 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1c1bb409-e03b-4ef3-bb0d-8b2ea94eaf65) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/location-capability.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a5f03b4369cd8467105750c61bc41eb4138475/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Flocation-capability.md) | 91 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7b384c2f-935a-4ac6-b1d0-d128654338a0) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/conversational-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a5f03b4369cd8467105750c61bc41eb4138475/msteams-platform%2Ftabs%2Fhow-to%2Fconversational-tabs.md) | 94 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/74c63729-b7a0-4a30-907c-391e30781194) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-channel-group-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a5f03b4369cd8467105750c61bc41eb4138475/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-channel-group-tab.md) | 91 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4b0c1a55-cb77-4a4d-b8cd-8ce00adc36d1) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a5f03b4369cd8467105750c61bc41eb4138475/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-personal-tab.md) | 93 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/31463772-19e7-4a25-9e5b-6388d5251b46) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-tab-pages/configuration-page.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a5f03b4369cd8467105750c61bc41eb4138475/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-tab-pages%2Fconfiguration-page.md) | 93 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/79783578-05a0-4bb7-8909-d39083dead12) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-tab-pages/content-page.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a5f03b4369cd8467105750c61bc41eb4138475/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-tab-pages%2Fcontent-page.md) | 91 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/79a81fba-6999-4a67-b86f-53cf50ff67b2) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/tabs-link-unfurling.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a5f03b4369cd8467105750c61bc41eb4138475/msteams-platform%2Ftabs%2Ftabs-link-unfurling.md) | 89 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fd7ff007-a637-44d4-a9ca-a0325494a6f7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by JoshuaPartlow** [comment]
PR created to fascilitate review, some edits expected still.

**Review by erikadoyle — COMMENTED** [review]
Looks good overall; just a minor suggestion

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/location-capability.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f9b576e158a180e3eb99352d72cca5fa09893ac7/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Flocation-capability.md) | 91 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d43aea1e-07d4-4fd3-a8e5-c7d4b925e794) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/conversational-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f9b576e158a180e3eb99352d72cca5fa09893ac7/msteams-platform%2Ftabs%2Fhow-to%2Fconversational-tabs.md) | 94 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b4efd24d-ff26-4d8c-854c-ef1caa080fd6) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-channel-group-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f9b576e158a180e3eb99352d72cca5fa09893ac7/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-channel-group-tab.md) | 91 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/adda51c8-f935-4ebc-a104-7b01be31dd39) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f9b576e158a180e3eb99352d72cca5fa09893ac7/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-personal-tab.md) | 93 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d8566790-6759-4043-9d13-be7f364f8bcb) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-tab-pages/configuration-page.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f9b576e158a180e3eb99352d72cca5fa09893ac7/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-tab-pages%2Fconfiguration-page.md) | 93 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/65232e93-244a-44b3-9c80-f671f114b88c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-tab-pages/content-page.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f9b576e158a180e3eb99352d72cca5fa09893ac7/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-tab-pages%2Fcontent-page.md) | 91 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f5085876-0555-452b-82e1-7de8409a884e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/tabs-link-unfurling.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f9b576e158a180e3eb99352d72cca5fa09893ac7/msteams-platform%2Ftabs%2Ftabs-link-unfurling.md) | 89 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c982f6a7-c7aa-4823-8c02-8478d33e6981) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/location-capability.md](https://github.com/MicrosoftDocs/msteams-docs/blob/aa15d5773aa773835fb648e4f2b304d219701b64/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Flocation-capability.md) | 91 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/108bcaa7-b279-41a8-9dea-9cb9f227656c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/conversational-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/aa15d5773aa773835fb648e4f2b304d219701b64/msteams-platform%2Ftabs%2Fhow-to%2Fconversational-tabs.md) | 94 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/94659f27-99c2-45c8-a534-592b41eb9edf) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-channel-group-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/aa15d5773aa773835fb648e4f2b304d219701b64/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-channel-group-tab.md) | 91 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9224fdbc-f258-42ca-9423-18a1912815c0) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/aa15d5773aa773835fb648e4f2b304d219701b64/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-personal-tab.md) | 93 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8d092835-156c-46d7-9810-8f1e43fb90df) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-tab-pages/configuration-page.md](https://github.com/MicrosoftDocs/msteams-docs/blob/aa15d5773aa773835fb648e4f2b304d219701b64/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-tab-pages%2Fconfiguration-page.md) | 93 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a9376765-1dca-4e96-80d7-2cbee2f477b9) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-tab-pages/content-page.md](https://github.com/MicrosoftDocs/msteams-docs/blob/aa15d5773aa773835fb648e4f2b304d219701b64/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-tab-pages%2Fcontent-page.md) | 91 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/242d545c-437b-4c98-9367-c69cdab58ba7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/tabs-link-unfurling.md](https://github.com/MicrosoftDocs/msteams-docs/blob/aa15d5773aa773835fb648e4f2b304d219701b64/msteams-platform%2Ftabs%2Ftabs-link-unfurling.md) | 92 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5e6c0115-c3b0-4ff9-9763-a52b1b3cd55e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/location-capability.md](https://github.com/MicrosoftDocs/msteams-docs/blob/32c842eb2b00e9470db7f3c19afa71fd8252975b/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Flocation-capability.md) | 91 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a23de206-28ec-4bb3-b77a-19c19de4ea89) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/conversational-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/32c842eb2b00e9470db7f3c19afa71fd8252975b/msteams-platform%2Ftabs%2Fhow-to%2Fconversational-tabs.md) | 94 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6a92c054-b09f-4d3f-8af8-305ead82763f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-channel-group-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/32c842eb2b00e9470db7f3c19afa71fd8252975b/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-channel-group-tab.md) | 91 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f53ec684-7b64-4179-9e69-4f3068bbdb94) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/32c842eb2b00e9470db7f3c19afa71fd8252975b/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-personal-tab.md) | 93 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/164c601d-a12a-4b7a-b818-51f4329fc745) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-tab-pages/configuration-page.md](https://github.com/MicrosoftDocs/msteams-docs/blob/32c842eb2b00e9470db7f3c19afa71fd8252975b/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-tab-pages%2Fconfiguration-page.md) | 93 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4c0029fa-90c1-4d21-8cda-faccc1021557) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-tab-pages/content-page.md](https://github.com/MicrosoftDocs/msteams-docs/blob/32c842eb2b00e9470db7f3c19afa71fd8252975b/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-tab-pages%2Fcontent-page.md) | 91 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1106d385-048e-4947-b077-77d1113e1251) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/tabs-link-unfurling.md](https://github.com/MicrosoftDocs/msteams-docs/blob/32c842eb2b00e9470db7f3c19afa71fd8252975b/msteams-platform%2Ftabs%2Ftabs-link-unfurling.md) | 92 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/be85296a-dfd7-4026-9e17-37ca4275078e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/tabs/tabs-link-unfurling.md [hunk 1]
```diff
@@ -82,17 +84,18 @@ Following is the process to invoke Stage View:
 * The bot responds with a `200` code.
 
 > [!NOTE]
+
 > On Teams mobile clients, invoking Stage View for apps distributed through the [Teams store](/platform/concepts/deploy-and-publish/apps-publish-overview.md) and not having a moblie-optimized experience opens the default web browser of the device. The browser opens the URL specified in the `websiteUrl` parameter of the `TabInfo` object.
 
 ## Invoke Stage View through deep link
 
-To invoke the Stage View through deep link from your tab, you must wrap the deep link URL in the `microsoftTeams.executeDeeplink(url)` API. The deep link can also be passed through an `OpenURL` action in the card.
+To invoke the Stage View through deep link from your tab, you must wrap the deep link URL in the `microsoftTeams.app.openLink(url)` API. The deep link can also be passed through an `OpenURL` action in the card.
```
[the remark this answers is not in the document]
Comment (v-abirade) [reply msteams-platform/tabs/tabs-link-unfurling.md:92]: @v-ksar, please update the comment.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/location-capability.md](https://github.com/MicrosoftDocs/msteams-docs/blob/bc9b2787b1fd3b12e1be67020755b7a6b6f24568/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Flocation-capability.md) | 91 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/35d04c14-16ff-46b7-8f1c-1710483406c3) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/conversational-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/bc9b2787b1fd3b12e1be67020755b7a6b6f24568/msteams-platform%2Ftabs%2Fhow-to%2Fconversational-tabs.md) | 94 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4da1bd1a-0faa-42ff-946b-f79f5806be50) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-channel-group-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/bc9b2787b1fd3b12e1be67020755b7a6b6f24568/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-channel-group-tab.md) | 91 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1f488b8b-b616-469f-9f85-5bc59604bdd6) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/bc9b2787b1fd3b12e1be67020755b7a6b6f24568/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-personal-tab.md) | 93 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/09a25e76-12bb-40af-a84f-a8342325ed97) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-tab-pages/configuration-page.md](https://github.com/MicrosoftDocs/msteams-docs/blob/bc9b2787b1fd3b12e1be67020755b7a6b6f24568/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-tab-pages%2Fconfiguration-page.md) | 93 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f29f1e40-11c0-41cd-8769-d3d096c8698f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-tab-pages/content-page.md](https://github.com/MicrosoftDocs/msteams-docs/blob/bc9b2787b1fd3b12e1be67020755b7a6b6f24568/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-tab-pages%2Fcontent-page.md) | 91 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4edacce0-6c0b-4233-96f8-ee7a2f4d8cd5) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/tabs-link-unfurling.md](https://github.com/MicrosoftDocs/msteams-docs/blob/bc9b2787b1fd3b12e1be67020755b7a6b6f24568/msteams-platform%2Ftabs%2Ftabs-link-unfurling.md) | 92 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3fe3a161-2d4d-4633-ba32-4681294028f4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-abirade — APPROVED** [review]
Approved.

**Pull request merged**